modules/installation-aws-permissions.adoc

```diff
@@ -99,16 +99,25 @@ If you use an existing VPC, your account does not require these permissions for
 * `elasticloadbalancing:ApplySecurityGroupsToLoadBalancer`
 * `elasticloadbalancing:AttachLoadBalancerToSubnets`
 * `elasticloadbalancing:ConfigureHealthCheck`
+* `elasticloadbalancing:CreateListener`
```

Comment:
Yeah, these were originally spearated based on the ELB versioning. The ELB v1 has more than  ELB v2 so @patrickdillon input would be valuable here for sure.

Reply:
This is the original PR (and it links to the BZ) that called for separating ELB & ELBv2. From what I can see in the bug, it does not make sense to split up the ELB permissions into two groups. Instead, imho, it would be better to include all ELB permissions under the same heading, and make sure that it includes all relevant permissions so no warnings are generated for either ELB or ELBv2 resources.

Reply:
Thanks. Updated. I remove the ELBv2 table but cross-referenced the deleted permissions against the ELBv1 table to ensure no replica permissions were deleted.

Reply:
> Instead, imho, it would be better to include all ELB permissions under the same heading

+1

Reply:
@patrickdillon I re-visited the original [Bug 1902748](https://bugzilla.redhat.com/show_bug.cgi?id=1902748), it‘s more of an AWS issue (or work as design?).

Some permissions are V1 or V2 exclusive, from user's perspective, if the policy contains a V2-only permission, e.g. `DescribeListeners`, a warning always   appears on the page:

```
V1+V2: AddTags
V1+V2: CreateLoadBalancer
V1+V2: DeleteLoadBalancer
V1+V2: DescribeLoadBalancerAttributes
V1+V2: DescribeLoadBalancers
V1+V2: DescribeTags
V1+V2: ModifyLoadBalancerAttributes
V1: ApplySecurityGroupsToLoadBalancer
V1: AttachLoadBalancerToSubnets
V1: ConfigureHealthCheck
V1: CreateLoadBalancerListeners
V1: DeregisterInstancesFromLoadBalancer
V1: DescribeInstanceHealth
V1: RegisterInstancesWithLoadBalancer
V1: SetLoadBalancerPoliciesOfListener
V2: CreateListener
V2: CreateTargetGroup
V2: DeregisterTargets
V2: DescribeListeners
V2: DescribeTargetGroupAttributes
V2: DescribeTargetHealth
V2: ModifyTargetGroup
V2: ModifyTargetGroupAttributes
V2: RegisterTargets
```

<img width="785" alt="image" src="https://github.com/openshift/openshift-docs/assets/61686698/2fb8d606-de3e-4aba-82f7-05975cef7912">

Reply:
Hi @yunjiang29 . I need to revert the changes and make sure ELB V1 and V2 are split out?

Reply:
I prefer to put them together actually, but my only concern is the user may ask the same question like [Bug 1902748](https://bugzilla.redhat.com/show_bug.cgi?id=1902748).

Two options may be applicable:
1) keep them unchanged
2) put all of them together in one section, and add a note to let user know that these permissions are not classified as ELB or ELBv2 permissions in OCP, but in AWS console

@patrickdillon do you have any better idea?

Reply:
Hi @yunjiang29 . I've updated the docs according to option 2 and Patrick's suggestion. OK to add a "LGTM"?